Update the game over message (#1)
HarisMG/introduction-to-github: README.md

  Do not use quotes on the <details> tag attributes.
-->

<!--step0

> All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.

Reviewer: ```suggestion
All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together as a team, which will enhance our skills 
```
